announcements/resources/schemas/dbscripts/sqlserver/comm-21.000-21.001.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2017-2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+UPDATE prop.properties
+SET value = 'secureOff'
+WHERE name = 'secure' AND value = 'false' AND
+    set IN (SELECT set FROM prop.propertysets WHERE category = 'messageBoardSettings');
```

Comment:
Haven't looked at the test failures, but given your comment I'm guessing you need to double-quote "SET" as it's a reserved word and I remember one of the DBs being picky

```suggestion
    "set" IN (SELECT "set" FROM prop.propertysets WHERE category = 'messageBoardSettings');
```

Reply:
Though the outages are only on sqlserver, tested this out and added quotes to both upgrade scripts for consistency 👍